public/index.html

```diff
@@ -15,6 +15,7 @@
       user's mobile device or desktop. See https://developers.google.com/web/fundamentals/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link rel="stylesheet" type="text/css" href="%PUBLIC_URL%/style.css">
```

Comment:
This is the public styling stuff, not something specific for one container/component. So, let us import it by the <link> tag in the entrance page index.html.